src/github.com/matrix-org/dendrite/roomserver/query/query.go

```diff
@@ -418,6 +418,108 @@ func (r *RoomserverQueryAPI) QueryServerAllowedToSeeEvent(
 	return nil
 }
 
+// QueryStateAndAuthChain implements api.RoomserverQueryAPI
+func (r *RoomserverQueryAPI) QueryStateAndAuthChain(
```

Comment:
doesn't this need calling from the router or something?

Reply:
Woops, fixed